### ISSUE TITLE
Warning: Failed prop type

### ISSUE THREAD

**Issue by vinit-poojary-au8** [yazaabed/react-tabs#27]
>  Warning: Failed prop type: Tabs: prop type `activeTab` is invalid; it must be a function, usually from the `prop-types` package, but received `object`.
>     in Tabs (at HomeBody.jsx:22)
>     in HomeBody (created by ConnectFunction)
>     in ConnectFunction (at Homepage.jsx:14)
>     in section (at Homepage.jsx:12)
>     in div (at Homepage.jsx:10)
>     in Homepage (created by Context.Consumer)
>     in Route (at App.js:12)
>     in Switch (at App.js:11)
>     in div (at App.js:10)
>     in App (at src/index.js:32)
>     in AuthIsLoaded (at src/index.js:31)
>     in Router (created by BrowserRouter)
>     in BrowserRouter (at src/index.js:30)
>     in ReactReduxFirebaseProvider (at src/index.js:29)
>     in Provider (at src/index.js:28)
>     in StrictMode (at src/index.js:27)

Okay I followed the demo but getting this warning

**Comment by yazaabed** [comment]
@vinit-poojary-au8 sorry for the late reply. I didn't get a notification that there is a new issue created. I hope I am not late. I will check it for you and deploy a fix. thank you for reporting this.

**Comment by squalvj** [comment]
yeah i followed the example aswell but got this error on typescript

`No overload matches this call.
  Overload 2 of 2, '(props: {}, context?: any): Tabs', gave the following error.
    Property 'title' is missing in type '{ id: string; }' but required in type 'TabInputs'.
  This JSX tag's 'children' prop expects a single child of type 'ReactDOM', but multiple children were provided.  TS2769`
